### PR TITLE
Doc: change placeholders<modulePath> to ~modulePath~

### PR DESCRIPTION
**Pull request by petersteneteg**
Fixes help links on windows

### CODE REVIEW

**Review by martinfalk — DISMISSED** [review]

File: modules/base/src/processors/convexhull2dprocessor.cpp [hunk 1]
```diff
@@ -77,7 +77,7 @@ const ProcessorInfo ConvexHull2DProcessor::processorInfo_{
     Tags::CPU,                           // Tags
     R"(Computes the convex hull of a 2D mesh.
     
-    Example Workspace: [core/convexhull.inv](file:///<basePath>/data/workspaces/convexhull.inv)
+    Example Workspace: [core/convexhull.inv](file:~basePath~/data/workspaces/convexhull.inv)
```
Comment (ResearchDaniel) [inline modules/base/src/processors/convexhull2dprocessor.cpp:80]: The // should remain?

**Review by ResearchDaniel — CHANGES_REQUESTED** [review]
Does not ~tilde~ cause a strikethrough?

File: modules/base/src/processors/convexhull2dprocessor.cpp (hunk 1 above)
Comment (petersteneteg) [reply modules/base/src/processors/convexhull2dprocessor.cpp:80]: No

**Comment by petersteneteg** [comment]
> Does not ~tilde~ cause a strikethrough?

No not in a link
~ are allowed in links without escaping < are not...

**Review by ResearchDaniel — DISMISSED** [review]

**Pull request merged**